content/300-guides/125-development-environment/100-editor-setup.mdx

```diff
@@ -40,7 +40,8 @@ You can install the official [Prisma VS Code extension](https://marketplace.visu
 
 ### Sublime Text
 
-- [sublimeText3](https://github.com/prisma/sublimeText3) provides syntax highlighting of the Prisma Schema Language.
+- [Prisma](https://packagecontrol.io/packages/Prisma) - For Sublime Text 3 & 4 - Provides syntax highlighting of the Prisma Schema Language. 
+- [LSP-prisma](https://packagecontrol.io/packages/LSP-prisma) - For Sublime Text 4 -  Language Server helper package for Prisma schema files that uses Prisma's Language Server to provide linting, error checking, formatting, autocompletion, renaming...
```

Comment:
```suggestion
- [LSP-prisma](https://packagecontrol.io/packages/LSP-prisma) - For Sublime Text 4 -  Language Server helper package for Prisma schema files that uses Prisma's Language Server to provide linting, error checking, formatting, autocompletion, renaming etc.
```

It may also be helpful to add that `LSP-prisma` requires the `Prisma` package to be installed. This is because the language server needs information on which files to attach itself, which is provided by the base scope (`source.prisma`) which the syntax package provides.